Jenkinsfile

```diff
@@ -378,19 +382,23 @@ pipeline {
                             source $HOME/google-cloud-sdk/path.bash.inc
                             gcloud auth activate-service-account --key-file $CLIENT_SECRET_FILE
                             gcloud config set project $GCP_PROJECT
-                            gcloud container clusters delete --zone $GKERegion $CLUSTER_NAME-basic $CLUSTER_NAME-scaling $CLUSTER_NAME-selfhealing $CLUSTER_NAME-backups $CLUSTER_NAME-bigdata $CLUSTER_NAME-upgrade || true
+                            gcloud container clusters list --format='csv[no-heading](name)' --filter $CLUSTER_NAME | xargs gcloud container clusters delete --zone $GKERegion --quiet || true
                             sudo docker system prune -fa
-
-                            sudo rm -rf $HOME/google-cloud-sdk
                         '''
                     }
                 }
             }
-            sh '''
-                sudo docker system prune -fa
-                sudo rm -rf ./*
-                sudo rm -rf $HOME/google-cloud-sdk
-            '''
+            withCredentials([string(credentialsId: 'GCP_PROJECT_ID', variable: 'GCP_PROJECT'), file(credentialsId: 'gcloud-key-file', variable: 'CLIENT_SECRET_FILE')]) {
```

Comment:
is any reason why we should have these commands twice?
the same commands in 381-385 lines